### PR TITLE
Update feature engineering/modelling

### PR DESCRIPTION
**Pull request by amadeuspzs**
# Summary

**What/Why**

This PR updates the modelling phase of the project in line with PR comments

**Your approach**

* Move LoS cap to data cleaning
* Update dataframe names to align
* Add comments on modelling approaches

**What to review**

* 03 Data cleaning
* 04 Feature Engineering
* 05a Modelling - regression
* 05b Modelling - regression fairness

# Checklists

This pull/merge request meets the following requirements:

- [X] code runs
- [X] [developments are ethical][data-ethics-framework]
- [X] developments are secure (no data or notebook output)
- [X] you have made proportionate checks that the code works correctly
- [X] [minimum usable documentation][agilemodeling] written in the `docs` folder

[agilemodeling]: http://agilemodeling.com/essays/documentLate.htm
[data-ethics-framework]: https://www.gov.uk/government/publications/data-ethics-framework

### CODE REVIEW

File: notebooks/05a Modelling - regression.ipynb [hunk 1]
```diff
@@ -389,15 +371,19 @@
    "source": [
     "## Train/test split\n",
     "\n",
-    "For model evaluation, we will hold back a 25% test set, and use cross-validation on the remaining 75% for all models until the final comparison is made."
+    "For model evaluation, we will hold back a 25% test set, and use cross-validation on the remaining 75% for all models until the final comparison is made.\n",
+    "\n",
+    "This helps improve the robustness of the models by reducing the risk of overfitting across the entire dataset.\n",
```
Comment (matthewcooper19) [inline notebooks/05a Modelling - regression.ipynb:376]: I would characterise the purpose of the test set as more being about enabling you to assess the performance of the model on a data set it has not be been exposed to before. Cross-validation checks for overfitting (as you have noted below)

**Review by matthewcooper19 — APPROVED** [review]

**Review by matthewcooper19 — APPROVED** [review]
